server/env/development.env

```diff
@@ -31,5 +31,6 @@ MONGO_PASSWORD=m0uw399xYkBSBZVr
 AWS_ACCESS_KEY=AKIAYBSARZCTOVGIBHXT
 AWS_SECRET_KEY=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
 AWS_REGION=us-east-2
+AWS_BUCKET=pizza-delivery-123j4ghj12h34

```

Comment:
Files like this needs to be in `.gitignore`. If you want to give like instructions to other developers, what needs to be in your `.env` - create not ignored example file `__.env` and copy all this keys without their values. Because it's sensitive data.